### ISSUE TITLE
non margin fell error

### ISSUE THREAD

**Issue by Geczy** [aseylys/KStock#4]
All I did was set TESTING to `False` in `KStock.py`

![image](https://user-images.githubusercontent.com/1036968/37259955-9d169f6c-2563-11e8-9d51-aa6e229449fb.png)

**Comment by Geczy** [comment]
After I press Ok it just keeps spamming it with the main app open

![image](https://user-images.githubusercontent.com/1036968/37259960-b00522ce-2563-11e8-9bae-57aa5c3e8a72.png)

**Comment by aseylys** [comment]
The non-margin fell below because you have $0 in Non-margin. It's not going to let you trade unless you have >$25000. That's an SEC rule.

**Comment by Geczy** [comment]
I have >$25k

The latest commit seems to have fixed it, but now it auto pauses and #2 is happening

![image](https://user-images.githubusercontent.com/1036968/37260421-35825d7c-2569-11e8-8780-e62e7c9f3ba4.png)

**Comment by aseylys** [comment]
No, you have $100k of equity. That's different. You need >$25k of Non-margin to actually trade, and the markets need to be open. If `TESTING=False` it will not trade **at all** after-hours or if you have <$25k non-margin.

**Comment by Geczy** [comment]
Will it auto resume when the stock market opens back up? Or do I have to press start manually? 

Also, how do I get >25k of non margin...? Sorry, new to the market

**Comment by aseylys** [comment]
You have to manually start it after 0930 EST, that's a safety net I personally wanted. 

Non-margin is how much cash you have on hand. You need to have >$25k not tied up in holdings to  day trade. The SEC uses that as, basically, leverage against you so they don't run the risk of you blowing through cash when day-trading. 

So to day trade, you need to deposit $25k into your account. That 25k will not be used to trade, it just sits there. Any cash you deposit after that will be used to trade. So if you deposit 30k, only 5k will be used to trade. The 25k will be used as leverage against you.

**Comment by Geczy** [comment]
I have 100k just sitting in my account not tied up in any stock. So is this a bug? 

![image](https://user-images.githubusercontent.com/1036968/37260730-f9f306fe-256c-11e8-866b-40cff810cc9d.png)

**Comment by Geczy** [comment]
And what kind of safety net does that grant? What I mean to ask is, how do you make the decision to *not* start the trading on a certain morning? Just so I know in case I shouldn't start trading

**Comment by Geczy** [comment]
>Withdrawal Holding Period
>Before you can initiate a withdrawal of your uninvested funds, your deposits must remain in your account for a minimum of 5 trading days. On the 6th day, those uninvested funds will go into your cash available for withdrawal. This withdrawal holding period is for anti-money laundering and risk management purposes.
>Please keep in mind that weekend and some US holidays don’t count as trading days.

Found what's happening, my cash is treated as AML Funds and the code is looking for `withdrawable_amount`

![image](https://user-images.githubusercontent.com/1036968/37284185-f2aa2846-25d0-11e8-8bb2-082d2573d3a4.png)

So I changed this line

```python
self.marginLabel.setText('%.2f' % (float(self.portfolio['withdrawable_amount'])))
```

To 

```python
totalCash = float(self.portfolio['unwithdrawable_deposits']) + float(self.portfolio['withdrawable_amount'])
self.marginLabel.setText('%.2f' % (totalCash))
```

And now it's fine. Let me know if you think this would cause an issue
